### PR TITLE
add support for for loops in fish shell

### PR DESCRIPTION
**Pull request by bdmorin**
## Description of changes (updated or new workflows)

Added a for loop template for fish shell

### CODE REVIEW

**Review by alokedesai — COMMENTED** [review]
Looks great, thank you for contributing! just two small comments on improvements to the actually YAML file we could make

File: specs/shell/shell_for_loop_fish.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+name: Shell for-loop
+command: "for {{variable}} in {{sequence}};\n  {{command}}\n; end"
```
[the remark this answers is not in the document]
Comment (bdmorin) [reply specs/shell/shell_for_loop_fish.yaml:3]: I didn't realize this was possible I just forked the for loop command for sh/zsh.

**Comment by alokedesai** [comment]
Feel free to re-request a review whenever the comments are addressed! Thank you again for contributing!

**Comment by alokedesai** [comment]
Closing this since there's been no activity. Feel free to reopen!

**Pull request closed without merging**